doc/examples/gradients_spheres.py

```diff
@@ -183,4 +183,3 @@
        anisotropic systems by magnetic resonance imaging, Magnetic Resonance
        in Medicine, vol 42, no 3, 515-525, 1999. 
 """
-
```

Comment:
I'm not sure removing the empty line will fix the rendering problem you noticed. Did you try rebuilding the example to see if it worked?

Reply:
No, I did not.  Hence calling it a 'blind attempt'.  I tried running `make html` and got an error at vtk, so I don't have the machinery to do a full make.

You're welcome to accept or decline any of the other changes and close the PR, but there appear now to be conflicts, as some of the sections I suggested changes to have been changed.

Please let me know if you would prefer that I not submit typo changes in the future without being able to run `make`.  If not, is it acceptable to submit them as issues?

Reply:
Hi,

yes, it seems that there are some conflicts. Most of them seem to be minor. Would you have the time to fix them?

Ideally, we prefer if people can run `make`, to ensure that the changes have the expected effect. It also helps the core maintainers (@skoudoro , @Garyfallidis ) if work is distributed. 

A PR is always better than an issue, but we understand in some cases if it is not possible.

I'm currently running the make on your branch to see if it works.

Reply:
Did the make succeed or fail when you ran it?